src/Stratis.Bitcoin/P2P/Peer/NetworkPeerServer.cs

```diff
@@ -177,7 +188,7 @@ public void Listen(int maxIncoming = 20)
             {
                 this.logger.LogDebug("Exception occurred: {0}", e.ToString());
             }
-            
+
```

Comment:
Inside this catch `catch (Exception e)` should be `this.logger.LogError` I would think.
And maybe even trigger a general cancellation of the node? @Aprogiena thoughts for our next skype call.

Reply:
> Inside this catch catch (Exception e) should be this.logger.LogError I would think.

I have changed this to logError 👍